packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -233,63 +252,113 @@ function restLevelPackage(model: RLCModel) {
+      packageInfo.devDependencies["playwright"] = "^1.41.2";
+      packageInfo.devDependencies["vitest"] = "^1.3.1";
+    }
     packageInfo.devDependencies["c8"] = "^8.0.0";
     packageInfo.devDependencies["source-map-support"] = "^0.5.9";
     packageInfo.devDependencies["ts-node"] = "^10.0.0";
-    packageInfo.scripts["test"] =
-      "npm run clean && npm run build:test && npm run unit-test";
-    packageInfo.scripts["test:node"] =
-      "npm run clean && npm run build:test && npm run unit-test:node";
-    packageInfo.scripts["test:browser"] =
-      "npm run clean && npm run build:test && npm run unit-test:browser";
-    packageInfo.scripts["build:browser"] =
-      "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1";
-    packageInfo.scripts["build:node"] =
-      "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1";
-    packageInfo.scripts["build:test"] = "tsc -p . && rollup -c 2>&1";
-    packageInfo.scripts["unit-test"] =
-      "npm run unit-test:node && npm run unit-test:browser";
-    packageInfo.scripts["unit-test:node"] =
-      // eslint-disable-next-line no-useless-escape
-      `cross-env TS_NODE_COMPILER_OPTIONS="{\\\"module\\\":\\\"commonjs\\\"}" mocha -r esm --require ts-node/register --timeout 1200000 --full-trace "test/{,!(browser)/**/}*.spec.ts"`;
-    packageInfo.scripts["unit-test:browser"] = "karma start --single-run";
-    packageInfo.scripts["integration-test:browser"] =
-      "karma start --single-run";
-    packageInfo.scripts["integration-test:node"] =
-      'nyc mocha -r esm --require source-map-support/register --timeout 5000000 --full-trace "dist-esm/test/{,!(browser)/**/}*.spec.js"';
-    packageInfo.scripts["integration-test"] =
-      "npm run integration-test:node && npm run integration-test:browser";
-
-    if (azureSdkForJs) {
-      packageInfo.scripts["build:test"] = "tsc -p . && dev-tool run bundle";
-      packageInfo.scripts["integration-test:browser"] =
-        "dev-tool run test:browser";
-      packageInfo.scripts["unit-test:browser"] = "dev-tool run test:browser";
+    if (model.options?.moduleKind === "cjs") {
+      packageInfo.scripts["test"] =
+        "npm run clean && npm run build:test && npm run unit-test";
+      packageInfo.scripts["test:node"] =
+        "npm run clean && npm run build:test && npm run unit-test:node";
+      packageInfo.scripts["test:browser"] =
+        "npm run clean && npm run build:test && npm run unit-test:browser";
+      packageInfo.scripts["unit-test"] =
+        "npm run unit-test:node && npm run unit-test:browser";
       packageInfo.scripts["unit-test:node"] =
-        "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'";
+        // eslint-disable-next-line no-useless-escape
+        `cross-env TS_NODE_COMPILER_OPTIONS="{\\\"module\\\":\\\"commonjs\\\"}" mocha -r esm --require ts-node/register --timeout 1200000 --full-trace "test/{,!(browser)/**/}*.spec.ts"`;
```

Comment:
We should leverage the work with `tsx` so that we can just call

`mocha --require tsx --timeout 1200000 --full-trace "test/{,!(browser)/**/}*.spec.ts"`